models/classes/ResultServerService.php

```diff
@@ -22,7 +22,17 @@
 interface ResultServerService {
     
     const SERVICE_ID = 'taoResultServer/resultservice';
-    
+
+    const CLASS_URI	= 'http://www.tao.lu/Ontologies/TAOResultServer.rdf#ResultServer';
+
+    const PROPERTY_MODEL = 'http://www.tao.lu/Ontologies/TAOResultServer.rdf#ResultServerModel';
+
+    const PROPERTY_HAS_MODEL ='http://www.tao.lu/Ontologies/TAOResultServer.rdf#hasResultServerModel';
+
+    const PROPERTY_MODEL_IMPL = 'http://www.tao.lu/Ontologies/TAOResultServer.rdf#implementation';
+
+    const INSTANCE_RESULT_SERVER = 'http://www.tao.lu/Ontologies/TAOResultServer.rdf#void';
```

Comment:
you should keep the void idea in the name. For now ze have 2 constants with the same name (in OntologyService) and it creates error